sprint/start_clean.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+SPRINT_BRANCH=8.7.x
```

Comment:
I still can't figure out a simple way to determine what the latest "development" branch is for core. One suggestion was to load packagist versions, but those aren't ordered so there'd still need to be logic for figuring out current Major version, latest minor version, and then on top of that translating to actual branch name.

Alternatively, in a follow-up, maybe could be stored as a file, and then that would make asserting the git branch  testable for any later refactor. At the moment it's probably not worth it to add a test of the git branch because of having to grep this file for this line to compare against`git rev-parse --abbrev-ref HEAD`.

Reply:
Thanks for looking! I guess it's not that  big a problem, but like the (former) problem with wiring in the latest docker-toolbox, things like this can bite in the future. We have an "OK' solution at this point.